### PR TITLE
Move data files to zivid data

### PR DESCRIPTION
**Pull request by SatjaSivcev**

### CODE REVIEW

File: source/hdev/Applications/Advanced/ObjectMatching/SurfaceMatchingFindModel.hdev [hunk 1]
```diff
@@ -52,7 +52,7 @@
 <c></c>
 <c>* Configuring HDR settings</c>
 <c>* Twinings tea box</c>
-<l>if (Filename = 'Twinings tea box')</l>
+<l>if (Filename = 'TwiningsTeaBox')</l>
 <l>    set_framegrabber_param (AcqHandle, 'MergedMultiFrameMode','HDR')</l>
```
Comment (chrisasc) [inline source/hdev/Applications/Advanced/ObjectMatching/SurfaceMatchingFindModel.hdev:56]: add space between',' and 'HDR', e.g. 'set_framegrabber_param (AcqHandle, 'MergedMultiFrameMode', 'HDR')'.
Need to do this with all others set_framegrabber_param (...) function calls as well.

**Comment by chrisasc** [comment]
LGTM. Regarding my comment about spacing, I suspect this is something we need to fix with all our Halcon samples.

File: source/hdev/Applications/Advanced/ObjectMatching/SurfaceMatchingFindModel.hdev (hunk 1 above)
Comment (SatjaSivcev) [reply source/hdev/Applications/Advanced/ObjectMatching/SurfaceMatchingFindModel.hdev]: Now I understand what made people make linters. Done.

**Comment by SatjaSivcev** [comment]
I've implemented the changes. Please approve if it is ready or if not, ready for the next round.

**Comment by SatjaSivcev** [comment]
> 
> 
> LGTM :)

Thanks. Try to remind me to squash, I proved to forget it ;)

**Pull request merged**